src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -5387,16 +5387,25 @@ s32 func_8083AD4C(PlayState* play, Player* this) {
     return Camera_ChangeMode(Play_GetCamera(play, CAM_ID_MAIN), cameraMode);
 }
 
-s32 func_8083ADD4(PlayState* play, Player* this) {
+/**
+ * If appropriate, setup action for performing a `csAction`
+ *
+ * @return  true if a `csAction` is started, false if not
+ */
+s32 Player_StartCsAction(PlayState* play, Player* this) {
```

Comment:
I'm trying to understand the difference between `Player_StartCsAction` and `Player_TryCsAction`, and the motivation for using one or the other. Both are used to setup `Player_Action_CsAction`, both can successfully or unsuccessfully start a `CsAction`. 

`Player_TryCsAction` tries to start the csAction through `if (this->unk_6AD == 3) {`, while `Player_StartCsAction` tries to start the csAction through `!Player_InBlockingCsMode(play, this)`.

What were your thoughts when differentiating these 2 functions

Reply:
There isnt really much of a "try" aspect for `Player_StartCsAction`. Except for a niche case with action 7, `Player_UpdateCommon` is going to set unk_6AD to 3 no questions asked. 

While `Player_TryCsAction` will make sure player is not already in a cutscene before trying

Reply:
Think the difference here is perspective on whos trying
Yeah `(this->unk_6AD == 3)` does mean the function itself is technically trying, but I view the interaction as a whole to be without restriction. The csAction "request" itself wont be denied when used through the interface of `Player_SetCsAction`, where it can be with `play->tryPlayerCsAction`